code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/filter/common/AbstractDefaultResourceManagerFilter.java

```diff
@@ -27,4 +29,17 @@ protected void validateDNSPatternForName(String name) {
         }
     }
 
+    protected void validateLinkPatternForName(String name) {
+        if (name != null) {
+            if(!LINK_NAME_PATTERN.matcher(name).matches()
```

Comment:
@ibuildthecloud true, only difference being the above regex does not allow a "/" in the link part - is that important to check or should I just remove this check?